scripts/scripts.js

```diff
@@ -15,7 +15,7 @@ import {
 } from './lib-franklin.js';
 
 const LCP_BLOCKS = ['hero']; // add your LCP blocks to the list
-const DELAYED_RESOURCES = 3000;
+const DELAYED_RESOURCES = 4500;
```

Comment:
not 100% sure it's safe to delay so long

Reply:
I ran each page a few times, while the first load was slower at about 2500ms for some pages, subsequent loads were all below 2000ms. So 3000ms is definitely safe.
Most of the topic pages were below 2000ms on the first load.

Reply:
I'm not sure I understood your point. I was saying that I'm not super sure if it is safe to delay the resources to `4500ms`. It would definitely improve the LHS, because there would be a more clear separation between load scripts and delay scripts and we can see that on the LHS numbers from the PR, but I don't know if that's acceptable for the functionality to load so late.